### DISCUSSION TITLE
MySQL integration?

### DISCUSSION THREAD

**Discussion by szemlicka** [advplyr/audiobookshelf#579] [General]
Hi,
I use audiobookshelf in a docker environment on a Synology NAS. Everytime I run a scan, the node use all CPU power and everything slow down. I don't know what is the bottlenek here.  But I think a MySQL DB could solve the problem. What do you think?

Is there a plan to integrate MySQL here as a DB and can this helps to make everthing quicker?

**Comment by advplyr** [comment]
The DB is not the bottleneck for scans. The bottleneck for scans is ffprobe running probes on your audio files. If you are using a remote file system this is going to be particularly bad since the disc reads are so slow.

There are no plans to use MySQL. If we improve the DB it will most likely be in the NoSql direction.

**Reply by szemlicka** [reply]
Thanks for the reply and to bring a little light on it. I use docker and no remote file system.
